extensions/opentelemetry/runtime/pom.xml

```diff
@@ -104,7 +100,7 @@
         </dependency>
         <dependency>
             <groupId>io.opentelemetry</groupId>
-            <artifactId>opentelemetry-api-events</artifactId>
+            <artifactId>opentelemetry-api-incubator</artifactId>
```

Comment:
what do you use this module for?

Reply:
We need to reset for tests the `GlobalEventLoggerProvider` otherwhise it throws an error when we execute dev mode. I remember that native would also failed without it. 
It's usage comes from the SDK Autoconfiguration.